test/function-block-shorthand.civet

```diff
@@ -920,6 +920,14 @@ describe "(op) shorthand", ->
     (1)
   """
 
+  testCase """
+    apply binary op with one argument in pipeline
+    ---
+    foo |> (+) x
+    ---
+    (x)(foo)
```

Comment:
Why is that the desired result?

Reply:
Oh, `(+) x` instantly unwraps, it’s not a section

Reply:
I'm not sure it's perfect, but the current behavior is that `(+)` can be followed by *any number* of arguments, and they get added together. In the special case of 1, it's not very useful, but maybe helpful in a setting where you're adding/removing arguments like so:

```js
(+)
  part1
  part2
//edit from/to:
(+)
  part1
  //part2
```

(similar to how we support empty blocks in `if` etc.)